presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -277,6 +262,43 @@ public static String startRemoteFunctionServer(String remoteFunctionServerBinary
         }
     }
 
+    public static NativeWorkerParameters getNativeQueryRunnerParameters()
+    {
+        Path prestoServerPath = Paths.get(Optional.ofNullable(System.getProperty("PRESTO_SERVER"))
+                        .orElse("_build/debug/presto_cpp/main/presto_server"))
+                .toAbsolutePath();
+        Path dataDirectory = Paths.get(Optional.ofNullable(System.getProperty("DATA_DIR"))
+                        .orElse("target/velox_data"))
+                .toAbsolutePath();
+        Optional<Integer> workerCount = Optional.ofNullable(System.getProperty("WORKER_COUNT")).map(Integer::parseInt);
+
+        assertTrue(Files.exists(prestoServerPath), format("Native worker binary at %s not found. Add -DPRESTO_SERVER=<path/to/presto_server> to your JVM arguments.", prestoServerPath));
+        log.info("Using PRESTO_SERVER binary at %s", prestoServerPath);
+
+        if (!Files.exists(dataDirectory)) {
+            assertTrue(dataDirectory.toFile().mkdirs());
+        }
+
+        assertTrue(Files.exists(dataDirectory), format("Data directory at %s is missing. Add -DDATA_DIR=<path/to/data> to your JVM arguments to specify the path", dataDirectory));
+        log.info("using DATA_DIR at %s", dataDirectory);
+
+        return new NativeWorkerParameters(prestoServerPath, dataDirectory, workerCount);
+    }
+
+    public static class NativeWorkerParameters
```

Comment:
Call this class NativeQueryRunnerParameters as well ?

Reply:
fixed